gatsby/content/projects/2018-04-24-xonotic.mdx

```diff
@@ -1,12 +1,17 @@
 ---
 layout: project
 title: xonotic
-categories:
- - client
-description: In-game chat
+categories: 
+ - other
```

Comment:
Updated to `other` so it shows up on http://localhost:8000/docs/projects/try-matrix-now under the other category

Currently, this page doesn't appear anywhere AFAIK